release/release.md

```diff
@@ -58,6 +58,13 @@ $ cd ${GOPATH}/src/github.com/kata-containers/packaging
 $ git pull
 ```
 1. Bump repositories
+
+   Trigger a [CI job][bump-job] to bump Kata
+
+   http://jenkins.katacontainers.io/blue/organizations/jenkins/release
```

Comment:
This link is the same as the `bump-job` one, so I think you can remove this line?